ceph_deploy/mon.py

```diff
@@ -233,10 +233,7 @@ def mon_add(args):
 
 def mon_create(args):
 
-    cfg = conf.ceph.load(args)
-    if not args.mon:
-        mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
-        args.mon = re.split(r'[,\s]+', mon_initial_members)
+    args.mon = get_mon_member(args)
```

Comment:
I think this line still needs to be in a "if not args.mon" block.  It should only be reading the mon_initial_members from the config file if args.mon is not already filled out.